src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -15,6 +15,7 @@
 #include <stddef.h>
 
 #include <algorithm>
+#include <cstdint>
```

Comment:
Why we are adding this?

Reply:
Because we use the `uint64_t` in this file. No?

Reply:
Ah, sure! :)